modules/ROOT/partials/access-neo4j.adoc

```diff
@@ -0,0 +1,22 @@
+By default, Neo4j Community Edition does not include graph tools such as visualization, data exploration, and monitoring.
+However, you can use the Neo4j Aura console to access these features for free.
+No subscription is required.
+
+. Sign up or log in to the link:https://console-preview.neo4j.io/self-managed[Aura Console].
+. On the *Instances* page, click the *Self-managed* tab and then *+ Add deployment* button.
+. Select *URL Connection*.
+. Provide a *Name* and *Connection URL*.
+If you have installed Neo4j locally on your system, you can connect to _\http://localhost:7474_.
+. Click the *Connect* dropdown to launch various graph tools such as *Query*, *Explore*, and *Dashboards*.
+. Type the username `neo4j` and your password or the default password `neo4j`.
+You will be prompted to change the latter upon first login.
++
+image::aura-add-deployment.png[width=800,alt=Aura add self-managed deployment]
+
+You are now connected and can use the Aura Console to run Cypher queries, visualize graphs, and optionally monitor your local Neo4j database in Neo4j Aura.
+For details, see link:https://neo4j.com/docs/getting-started/#_work_with_data[Get started with Neo4j].
+
+Alternatively, you can use the Neo4j Browser, a web-based user interface for interacting with Neo4j that is included with the Neo4j installation.
+To access the Neo4j Browser, open a web browser and navigate to _\http://localhost:7474_.
```

Comment:
This contradicts what was said in line 1:
> By default, Neo4j Community Edition does not include graph tools such as visualization, data exploration, and monitoring.

Maybe there are special business reasons, but I would first mention the old way (ie using browser by going to localhost:7474) and then the magic new way through aura. The path of least resistance between installing neo4j and having a working neo4j is by going directly to localhost:7474